week-1-practise/level-1/4-logic-error-1.js

```diff
@@ -1,7 +1,7 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  return word;
 }
```

Comment:
you are only returning word  this means it will be the same text with spaces, so use the trim method

Reply:
I have changed to return word.trim();